wallet-sdk/src/main/kotlin/org/stellar/walletsdk/Wallet.kt

```diff
@@ -479,4 +480,41 @@ class Wallet(
       throw e
     }
   }
+
+  /**
+   * Get account operations for the specified Stellar address.
+   *
+   * @param accountAddress Stellar address of the account
+   * @param limit optional how many operations to fetch, maximum is 200, default is 10
+   * @param order optional data order, ascending or descending, defaults to descending
+   * @param cursor optional cursor to specify a starting point
+   * @param includeFailed optional flag to include failed operations, defaults to false
+   *
+   * @return a list of formatted operations
+   *
+   * @throws [OperationsLimitExceededException] when maximum limit of 200 is exceeded
+   */
+  suspend fun getHistory(
+    accountAddress: String,
+    limit: Int? = null,
+    order: Order? = null,
+    cursor: String? = null,
+    includeFailed: Boolean? = null
+  ): List<WalletOperation<OperationResponse>> {
+    if (limit != null && limit > 200) {
+      throw OperationsLimitExceededException()
+    }
+
+    return server
+      .operations()
+      .forAccount(accountAddress)
+      .limit(limit ?: 10)
```

Comment:
This comment
https://github.com/stellar/kotlin-wallet-sdk/pull/28/files#r1035228582
Is applied here as well (because horizon has it's defaults, so duplicating them here is not a good idea, they may be changed in the future. Instead, don't set them)

Reply:
Ideally, I'd like to have the same limit default everywhere. For anchors, limit is optional and might not work at all if provided. I'm good either way. We can check with others what they think. :nerd_face:

Reply:
I agree. My point is that if system has defaults (like Horizon), it's better to not re-define them in our code. (unless we want to)